file/builder_test.go

```diff
@@ -2439,3 +2440,427 @@ func Test_stateBuilder_fillPluginConfig(t *testing.T) {
+											"default": []interface{}{},
+											"elements": map[string]interface{}{
+												"type": "string",
+											},
+											"required": true,
+											"type":     "array",
+										},
+									},
+									map[string]interface{}{
+										"uri": map[string]string{
+											"type": "string",
+										},
+									},
+								},
+								"required": true,
+								"type":     "record",
+							},
+						},
+						map[string]interface{}{
+							"add": map[string]interface{}{
+								"fields": []interface{}{
+									map[string]interface{}{
+										"body": map[string]interface{}{
+											"default": []interface{}{},
+											"elements": map[string]interface{}{
+												"type": "string",
+											},
+											"required": true,
+											"type":     "array",
+										},
+									},
+									map[string]interface{}{
+										"headers": map[string]interface{}{
+											"default": []interface{}{},
+											"elements": map[string]interface{}{
+												"match": "^[^:]+:.*$",
+												"type":  "string",
+											},
+											"required": true,
+											"type":     "array",
+										},
+									},
+									map[string]interface{}{
+										"querystring": map[string]interface{}{
+											"default": []interface{}{},
+											"elements": map[string]interface{}{
+												"type": "string",
+											},
+											"required": true,
+											"type":     "array",
+										},
+									},
+								},
+								"required": true,
+								"type":     "record",
+							},
+						},
+						map[string]interface{}{
+							"append": map[string]interface{}{
+								"fields": []interface{}{
+									map[string]interface{}{
+										"body": map[string]interface{}{
+											"default": []interface{}{},
+											"elements": map[string]interface{}{
+												"type": "string",
+											},
+											"required": true,
+											"type":     "array",
+										},
+									},
+									map[string]interface{}{
+										"headers": map[string]interface{}{
+											"default": []interface{}{},
+											"elements": map[string]interface{}{
+												"match": "^[^:]+:.*$",
+												"type":  "string",
+											},
+											"required": true,
+											"type":     "array",
+										},
+									},
+									map[string]interface{}{
+										"querystring": map[string]interface{}{
+											"default": []interface{}{},
+											"elements": map[string]interface{}{
+												"type": "string",
+											},
+											"required": true,
+											"type":     "array",
+										},
+									},
+								},
+								"required": true,
+								"type":     "record",
+							},
+						},
+					},
+					"required": true,
+					"type":     "record",
+				},
+			},
+		},
+	}
+)
+
+func TestFillPluginDefaults(T *testing.T) {
+	tests := []struct {
+		name     string
+		plugin   *kong.Plugin
+		schema   utils.Schema
+		expected *kong.Plugin
+	}{
```

Comment:
Remove these tests. They are now part of go-kong.

Reply:
done